lib/instance_summary.rb

```diff
@@ -0,0 +1,287 @@
+require 'text-table'
+require 'thread'
+require 'thwait'
+
+module BushSlicer
+  class InstanceSummary
```

Comment:
Could you add some description about what this class is doing? I looks like belonging to `tools/lib`.